### PR TITLE
clean travis conf

### PR DESCRIPTION
**Pull request by Thiblef**

### CODE REVIEW

**Comment by itkg-alavieille** [comment]
:+1:

**Comment by alavieille** [comment]
:+1:

File: .travis.php.ini [hunk 1]
```diff
@@ -1,3 +1 @@
-extension = "mongo.so"
 extension = "apc.so"
```
Comment (nicolasThal) [inline .travis.php.ini:1]: I think that apc is not required anymore, you can also suppress it

File: .travis.yml [hunk 2]
```diff
@@ -8,60 +8,24 @@ sudo: false
 
 matrix:
   include:
-    - env: MONGOPHP=3.0.5.4
+    - env: PHP=5.4
```
Comment (nicolasThal) [inline .travis.yml:11]: As the only thing moving in the matrix you could avoid this usage and put only : 

``` yaml
php:
  - 5.4
  - 5.5
  - 5.6

addons:
  apt:
    packages:
      -libap-tools
```

**Comment by Thiblef** [comment]
https://github.com/open-orchestra/open-orchestra-media-demo/pull/43

**Pull request merged**